src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -203,6 +248,30 @@ private static String getTextOrEmptyString(TextField textField) {
     return textField.getText() != null ? textField.getText().trim() : "";
   }
 
+  private static String convertConfigWinnerElectionModeToGuiText(ContestConfig config) {
+    switch (config.getWinnerElectionMode().toString()) {
+      case "standard" -> {
+        return config.getNumberOfWinners() > 1 ? "Multi-winner allow multiple winners per round"
+            : "Single-winner majority determines winner";
+      }
+      case "singleSeatContinueUntilTwoCandidatesRemain" -> {
+        return "Single-winner majority determines winner";
+      }
+      case "multiSeatAllowOnlyOneWinnerPerRound" -> {
+        return "Multi-winner allow only one winner per round";
+      }
+      case "multiSeatBottomsUp" -> {
+        return config.getNumberOfWinners() == 0
+            || config.getMultiSeatBottomsUpPercentageThreshold() != null
+            ? "Bottoms-up using percentage threshold" : "Bottoms-up";
+      }
+      case "multiSeatSequentialWinnerTakesAll" -> {
+        return "Multi-pass IRV";
```

Comment:
can we define constants for these human-readable strings?

Reply:
Absolutely, I thought of this immediately after I sent the PR.